src/data/resolvers/transactions.ts

```diff
@@ -211,3 +220,67 @@ export const getColonyUnclaimedTransfers = async (
 
   return transfers.filter(notUndefined);
 };
+
+export const transactionResolvers = ({
+  colonyManager: { networkClient },
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async transaction(_, { transactionHash, colonyAddress }) {
+      const { provider } = networkClient;
+      const colonyClient = await networkClient.getColonyClient(colonyAddress);
+
+      /*
+       * Try to get the transaction receipt. If the transaction is mined, you'll
+       * get a return from this call, otherwise, it's null.
+       */
+      const transactionReceipt = await provider.getTransactionReceipt(
+        transactionHash,
+      );
+
+      if (transactionReceipt) {
+        const {
+          transactionHash: hash,
+          from,
+          to,
+          status,
+          logs,
+        } = transactionReceipt;
+        const events = logs
+          ?.map((log) => colonyClient.interface.parseLog(log))
+          /*
+           * If the above parser find events that are not part of the colony client
+           * it will return them as `null` so we filter them out
+           */
+          .filter((log) => !!log)
+          .map(({ name, values, topic }) => ({ name, values, topic }));
+        return {
```

Comment:
A crap. We also need to return blockTime in here so we can get an accurate date.

I'll put this in tommorow